packages/reporter/src/header/stats.tsx

```diff
@@ -12,21 +14,35 @@ interface Props {
 
 const Stats = observer(({ stats }: Props) => (
   <ul aria-label='Stats' className='stats'>
-    <li className='passed'>
-      <i aria-hidden="true" className='fas fa-check'></i>
-      <span className='visually-hidden'>Passed:</span>
-      <span className='num'>{count(stats.numPassed)}</span>
+    <li>
+      <button className='passed' onClick={() => runnablesStore.setFilter('passed')} disabled={runnablesStore.activeFilter === 'passed'}>
+        <i aria-hidden="true" className='fas fa-check'></i>
+        <span className='visually-hidden'>Passed:</span>
+        <span className='num'>{count(stats.numPassed)}</span>
+      </button>
     </li>
-    <li className='failed'>
-      <i aria-hidden="true" className='fas fa-times'></i>
-      <span className='visually-hidden'>Failed:</span>
-      <span className='num'>{count(stats.numFailed)}</span>
+    <li>
+      <button onClick={() => runnablesStore.setFilter('failed')} className="failed" disabled={runnablesStore.activeFilter === 'failed'}>
+        <i aria-hidden="true" className='fas fa-times'></i>
+        <span className='visually-hidden'>Failed:</span>
+        <span className='num'>{count(stats.numFailed)}</span>
+      </button>
     </li>
-    <li className='pending'>
-      <i aria-hidden="true" className='fas fa-circle-notch'></i>
-      <span className='visually-hidden'>Pending:</span>
-      <span className='num'>{count(stats.numPending)}</span>
+    <li>
+      <button className='pending' onClick={() => runnablesStore.setFilter('processing')} disabled={runnablesStore.activeFilter === 'processing'}>
+        <i aria-hidden="true" className='fas fa-circle-notch'></i>
+        <span className='visually-hidden'>Pending:</span>
+        <span className='num'>{count(stats.numPending)}</span>
+      </button>
     </li>
+    {runnablesStore.activeFilter !== null && <li>
+      <Tooltip placement='bottom' title={<p>Clear the filter</p>}>
```

Comment:
ℹ️ Since it's a new icon appearing on the screen, I find it useful to provide a Tooltip so that user can get used to it 😊